k8s/vagrant/README.md

```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
 ubuntu-host:~/k8s-demo$ vagrant up
```

Comment:
..... Vagrant Up

Reply:
the command is *vagrant up* -- with lowercase "v". Leaving it as is..